package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac

```diff
@@ -76,6 +76,7 @@ local primary_addrs = {
 			'ocedo,raccoon',
 			'tplink,archer-a7-v5',
 			'tplink,archer-c2-v3',
+			'tplink,archer-c7-v5',
```

Comment:
Correct MAC is assigned without this, just tested this quick as the v2 doesnt need this, too.

Reply:
pushed it for you; resolved in d38c4cbf71ea8503453b3712b28e645779be3a1e.
Not sure if this is not a bug then.

Reply:
Optimally, all devices should have the correct label MAC device set in the Device Tree, making the list in Gluon obsolete.